packages/DraftEditor/src/DraftPlugins/CustomPlugins/LinkControl/LinkControl.tsx

```diff
@@ -46,6 +46,18 @@ const LinkControl = ({
     }
   };
 
+  const getUrl = (str: string) => {
+    if (
+      str.includes(".") &&
+      !str.match(/^http[s]?:\/\//) &&
+      !str.includes("mailto")
+    ) {
+      return `http://${str}`;
+    }
+
+    return url;
```

Comment:
just checking - you want to return the url from the state, not the `str` that you sent to getUrl function?

Reply:
You're right, fixed